Rakefile

```
@@ -217,6 +217,32 @@ namespace :release do
        "Groonga #{version}!!!")
     sh("git", "push", "origin", "v#{version}")
   end
+
+  desc "Post release announces"
+  task :announce do
+    latest_news = Dir.glob("doc/source/news/*.*").max do |a, b|
+      File.basename(a).to_i - File.basename(b).to_i
+    end
+    latest_release_note = File.read(latest_news).split(/^## /)[1]
+    latest_release_note_version = latest_release_note.lines.first[/[\d.]+/]
+    if latest_release_note_version != version
+      raise "release note isn't written"
+    end
+    latest_release_summary =
+      latest_release_note[/^[^\n]+\n\n(.+?)(?=\n###|\z)/m, 1]
```

Can we simplify this? In general, we should not use positive lookahead as much as possible.

```suggestion
      latest_release_note.split(/^### /, 2)[0].gsub(/\A.*\n/, "").strip
```